src/test/java/domain/ProfitTest.java

```diff
@@ -0,0 +1,79 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+public class ProfitTest {
+
+    Map<Rank, Integer> calculateResult = new LinkedHashMap<>();
+
+    @BeforeEach
+    void init() {
+        for (Rank rank : Rank.values()) {
+            calculateResult.put(rank, 0);
+        }
+    }
+
+    @Nested
+    @DisplayName("단일 당첨 케이스")
+    class singleProfit {
+
+        @DisplayName("구매 금액이 1000, 1등이 당첨되었을 경우, 2000000.0의 수익률이 결과로 나와야 한다")
+        @Test
+        void profit_test_1() {
+            assertValidateProfit(1000, List.of(Rank.FIRST), 2000000.0);
+        }
+
+        @DisplayName("구매 금액이 5000, 5등이 당첨되었을 경우, 1.0의 수익률이 결과로 나와야 한다")
+        @Test
+        void profit_test_2() {
+            assertValidateProfit(5000, List.of(Rank.FIFTH), 1.0);
+        }
+
+        @DisplayName("구매 금액이 1000, 미당첨일 경우, 수익률은 0.0이어야 한다.")
+        @Test
+        void profit_test_3() {
+            assertValidateProfit(1000, List.of(Rank.NONE), 0.0);
+        }
+    }
+
+    @Nested
+    @DisplayName("중복 당첨 케이스")
+    class profits {
```

Comment:
```suggestion
    class Profits {
```

클래스명은 대문자로 시작하는 것이 자바 코드 컨벤션이죠!
테스트코드에서 소문자로 시작하는 클래스명이 많은데 전체적으로 한번 확인해주세요. 🙂

Reply:
아이고 내부 클래스 선언할 때 전혀 생각치도 못 하고 있었네요... 수정하겠습니다!